packages/manager/src/features/Linodes/LinodesLanding/LinodeRow/LinodeRow.tsx

```diff
@@ -163,6 +177,17 @@ export const LinodeRow = (props: Props) => {
             <RegionIndicator region={region} />
           </TableCell>
         </Hidden>
+        {flags.vpc && (
+          <Hidden smDown>
+            <TableCell noWrap>
+              {vpcLabel && (vpcId === 0 || vpcId) && (
```

Comment:
We can simplify this!

```suggestion
              {vpcLabel && (
```